integrationtests/server-integration/server-integration-commons/src/test/java/org/infinispan/test/integration/persistence/jdbc/util/TableManipulation.java

```diff
@@ -31,9 +31,13 @@ public class TableManipulation implements AutoCloseable {
    private static final String TABLE_NAME_PREFIX = "tbl";
    private final String tableName;
 
-   public TableManipulation(String cacheName, PooledConnectionFactoryConfigurationBuilder persistenceConfiguration, ConfigurationBuilder configurationBuilder) {
-      this.persistenceConfiguration = persistenceConfiguration;
-      this.tableName = new TableName(DEFAULT_IDENTIFIER_QUOTE_STRING, TABLE_NAME_PREFIX, cacheName).getName();
+   public TableManipulation(String cacheName, JdbcConfigurationUtil jdbcConfigurationUtil) {
+      this.persistenceConfiguration = jdbcConfigurationUtil.getPersistenceConfiguration();
+      if(jdbcConfigurationUtil.driverClass.contains("oracle") || jdbcConfigurationUtil.driverClass.contains("db2")) {
```

Comment:
this is confusing to me.
* `AbstractTableManager` is using `TableName` to compute the table name
* The tests in `server/tests` are using `String.format`
* And here is using a mix...

I'm wondering which one is correct :confused:

Reply:
TBH I am not sure, @gustavolira may be able to answer better. However, I can resurrect some changes I had done to retrieve the table name directly from the TableManager. Updating shortly.

Reply:
I have updated to use the TableName directly instead.